scripts/exglobal_prep_ocean_obs.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+# exglobal_prep_ocean_obs.py
+# Pepares observations for coupled ocean DA
+import os
+from wxflow import YAMLFile
+import prep_marine_obs
+
+OBS_YAML = os.getenv('OBS_YAML')
+print("OBS_YAML:", OBS_YAML)
+
+data = YAMLFile(OBS_YAML)
+print(data)
+
+for observer in data['observers']:
+    obs_source_name = observer['obs space']['name']
+    print(obs_source_name)
```

Comment:
use logging instead of print if you really want to keep the output. Something like this:
```python
logging.info(f"obs source: {obs_source_name}")
```

Reply:
done